spec/stack.js

```diff
@@ -1,22 +1,53 @@
-import chai, { expect } from 'chai'
+import chai, { expect, assert } from 'chai'
 import chaiChange from 'chai-change'
 import Stack from '../src/stack'
 
 chai.use(chaiChange)
 
 describe('Stack', () => {
   'use strict'
+  let myStack
+  beforeEach( () => {
+    myStack = new Stack()
+  })
 
   it('exists', () => {
     expect(Stack).to.be.a('function')
   })
 
-  context('push()', () => {
+  context('#push()', () => {
     it('pushes an element to the top of the stack.', () => {
-      const myStack = new Stack()
-
       expect(() => myStack.push('foo'))
         .to.alter(() => myStack.length(), { from: 0, to: 1 })
```

Comment:
Consider testing additional properties (if not for Stack, then on future data structures). For instance, push might test:

```js
it('changes the top element to the value pushed.', () => {
      const myStack = new Stack()

      expect(() => myStack.push('bar'))
        .to.alter(() => myStack.peek(), { from: null, to: 'bar' })
    })
```

I was proud of the (tests we wrote)[https://github.com/EthanJStark/core-data-structures], if you ever wanna take a peek.